src/generic_core/social.ts

```diff
@@ -82,16 +82,10 @@ export function initializeNetworks() :void {
  */
 export function getNetwork(networkName :string, userId :string) :social.Network {
   if (!(networkName in networks)) {
-    log.warn('Network does not exist', networkName);
-    return null;
+    throw new Error('unknown network ' + networkName);
```

Comment:
You may also use template strings:

    throw new Error(`unknown network ${networkName}`);


Up to you.